oe_multilingual.install

```diff
@@ -36,29 +36,31 @@ function oe_multilingual_install() {
       '/node/*/translations',
     ])->save();
 
+  // Make sure that English language prefix is set to "en".
+  \Drupal::configFactory()
+    ->getEditable('language.negotiation')
+    ->set('url.prefixes.en', 'en')
+    ->save();
```

Comment:
The other language settings are not coming from core but from our module's `./config/install` so we "control them" so to say. The English language is the only one coming from core, that's why it's special handed.

About the site builder messing with configuration: we won't have this case since the OpenEuropa components are meant to be enabled at the beginning of the development process, the site configuration will be then exported in the site's `config/sync` and committed to the site repo.

I do agree, however, that this might generate confusion, we should either document this or have a talk with the team about this, there is also a ticket about configuration management, we could deal with this in that ticket's scope.